test/unit/TablelandTablesProxy.ts

```diff
@@ -84,7 +101,34 @@ describe("TablelandTablesProxy", function () {
     const Factory2 = await ethers.getContractFactory(
       "TestTablelandTablesUpgrade"
     );
-    const tables2 = await update(tables1, Factory2);
+    const tables2 = await upgrade(tables1, Factory2);
+    const impl2 = await upgrades.erc1967.getImplementationAddress(
+      tables2.address
+    );
+
+    // Test implementation was upgraded
+    expect(impl1).to.not.equal(impl2);
+    expect(tables1.address).to.equal(tables2.address);
+
+    // Test storage has not changed
+    expect(await tables2.balanceOf(owner.address)).to.equal(BigNumber.from(1));
+  });
+
+  it.only("Should allow implementation to be upgraded to constructor with _disableInitializers() call", async function () {
```

Comment:
new test:
- tests that we can upgrade our current live contracts to ones that have the new constructor w/o effecting storage.
- tests that we can upgrade from a contract w/ the constructor to new implementations w/o effecting storage.